assets/_sass/_editor.scss

```diff
@@ -28,13 +28,26 @@ body::after{
     display: none;
 }
 
-.ce-toolbar__actions {
-    right: 100%;
-    top: -2px;
+@media (max-width: 768px) {
+    .ce-toolbar__actions {
+        right: 0px !important; // Place the tune menu on the right side of the screen when it's narrow
```

Comment:
What CSS is the `!important` overwriting, that we need to use `important`? In my experience, it should be a last resort.

Reply:
It is a last resort. EditorJS messes with styles through JS. I need to prevent this to fix these issues.

Reply:
Ah okay, 100%.